src/Bridges/bridge_optimizer.jl

```diff
@@ -28,6 +28,11 @@ instead of passing it as is to its internal model.
 
 Return a `Bool` indicating whether `b` tries to bridge constrained variables in
 `S` instead of passing it as is to its internal model.
+
+    is_bridged(b::AbstractBridgeOptimizer, F::Type{<:MOI.AbstractFunction})::Bool
+
+Return a `Bool` indicating whether `b` tries to bridge objective functions of
```

Comment:
`tries to bridge` or `bridges`?

Reply:
For `LazyBridgeOptimizer`, `is_bridged(b, ...) = !supports_constraint(b.model, ...)` so it does not mean we can bridge it, it just mean that we should try since the inner model does not support it.